PSReadLine/Changes.txt

```diff
@@ -1,3 +1,26 @@
+### Version 2.1.0-beta1
+
+Pre-release notes:
+
+#### Prototype to support the fish-like auto-suggestion in PSReadLine
+
+Currently:
+
+- The only suggestion source is the PSReadLine history file.
+  The ultimate goal is to call into a prediction API in PowerShell engine that is powered by a prediction plugin.
```

Comment:
>The ultimate goal is to call into a prediction API in PowerShell engine that is powered by a prediction plugin. [](start = 2, length = 111)

I don't think future plans belong in the change log.

Reply:
Actually, I don't think most of this section belongs in the change log - documentation should live elsewhere.

---
In reply to: [420564331](https://github.com/PowerShell/PSReadLine/pull/1496#discussion_r420564331) [](ancestors = 420564331)

Reply:
I will remove this section.